ingestion/src/metadata/pii/scanners/ner_scanner.py

```diff
@@ -44,7 +44,7 @@ class NLPEngineModel(BaseModel):
     """Required to pass the nlp_engine as {"lang_code": "en", "model_name": "en_core_web_lg"}"""
 
     lang_code: str
-    model_name: str
+    nlp_model_name: str
```

Comment:
to fix this warning

![image](https://github.com/user-attachments/assets/dc7dd5fa-20b9-4051-b672-fdbcd1d00f62)